schemas/fingerprint-server-api.yaml

```diff
@@ -719,6 +739,48 @@ components:
           required:
             - code
             - message
+    Error400DeleteVisitsResponse:
+      type: object
+      additionalProperties: false
+      properties:
+        error:
+          type: 'object'
+          additionalProperties: false
+          title: ErrorDeleteVisitsResponseError
+          properties:
+            code:
+              type: string
+              description: >
+                Error code:
+                * `RequestCannotBeParsed` - The visitor ID is invalid.
+                * `RequestCannotBeParsed` - The visitor ID is required.
```

Comment:
Is this intentionally `RequestCannotBeParsed` twice?

Reply:
in both situation it's the same error code, with a different message
do you want to have a single message ?

Reply:
```suggestion
                * `RequestCannotBeParsed` - The visitor ID is missing or in the wrong format.
```